centos_install_requirements.sh

```diff
@@ -27,7 +30,7 @@ fi
 # Install required packages
 # python-{requests,setuptools} required for tripleo-repos install
 sudo yum -y install \
-  python-pip \
```

Comment:
Is python-pip installed in RHEL7 and RHEL8 by default?

Reply:
No it's not, but it will be installed via packages_installation before we attempt any pip installs, so I removed it here and added python3-pip for RHEL8 (packages_installation already defined python-pip for centos/RHEL7 but that won't work on RHEL8)